opennms-doc/guide-admin/src/asciidoc/text/enlinkd/layer-2/bridge-discovery.adoc

```diff
@@ -0,0 +1,116 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../../images
+
+==== Transparent Bridge Discovery
+
+Discovering _Layer 2_ network links using the _Bridge Forwarding_ table requires a special algorithm.
+To discover _Links_ an algorithm based on a scientific paper with the title link:http://cs-pub.bu.edu/groups/nrg/readinglist/lowekamp-sigcomm01.pdf[Topology Discovery for Large Ethernet Networks] is implemented.
+The gathered information is used to classify _Links_ in _macLink_ and _bridgeLink_.
+A _macLink_ represents a _Link_ between a workstation or server identified by a mac address.
+A _bridgeLink_ is a _connection_ between backbone ports.
+
+Transparent bridging is not loop free so if you have loops you have to enable the spanning tree protocol that will detect loops and again will put some ports in a _blocking_ state to avoid loops.
+To get links it is necessary to perform some calculations that let us define the _Links_.
+The following _MIBS_ must be supported by the _SNMP agent_ to allow _Transparent Bridge Discovery_.
```

Comment:
Are all the OIDs really mandatory? ie: Will transparent bridge discovery fails should any single OID from this list be missing? (I guess the answer is "no" since "vtpVersion" doesn't seem mandatory. But, what about the other OIDs?)